src/targets/gpu/kernels/include/migraphx/kernels/math.hpp

```diff
@@ -101,6 +101,7 @@ MIGRAPHX_DEVICE_MATH(erf, ::erf)
 MIGRAPHX_DEVICE_MATH(exp, ::exp)
 MIGRAPHX_DEVICE_MATH(floor, ::floor)
 MIGRAPHX_DEVICE_MATH(isnan, ::isnan)
+MIGRAPHX_DEVICE_MATH(isinf, ::isinf)
```

Comment:
For the GPU implementation need to add tests inside `test_verify.cpp`. Can you add a few please ?

Reply:
I've added those test cases